code/src/components/Form.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef } from 'react'
+import {RadioForm} from 'components/Radio'
+import {Text} from 'components/Text'
+
+export const values = {name: "", room: "", city: "", parking: "no"}
+
+export const Form = () => {
+  const inputRef = useRef()
+  const [done, setDone] = useState(false)
+  
+  // var hasName = (name === 'true') ? 'Y' :'N';
+  const handleSubmit = (evt) => {
```

Comment:
I guess "evt" is a short for "event?"
```suggestion
  const handleSubmit = (event) => {
```